libc/test/include/CMakeLists.txt

```diff
@@ -33,3 +33,16 @@ if(LLVM_LIBC_FULL_BUILD AND libc.include.stdbit IN_LIST TARGET_PUBLIC_HEADERS)
       # to.
   )
 endif()
+
+# Similar situation applies to stdckdint_test.
+if(LLVM_LIBC_FULL_BUILD)
```

Comment:
nit: Can you double check that this test runs fine with a clean overlay mode build?  It does not depend on generated headers any more, since `libc.include.llvm-libc-macros.stdckint_macros` is a normal header library target.

Reply:
Tested. It works. I think I will just remove the guards then.